lib/web_of_science/harvester.rb

```diff
@@ -5,6 +5,7 @@ module WebOfScience
   # This class is responsible for processing WebOfScience API response data
   # to integrate it into the application data models.
   class Harvester
+    delegate :logger, to: :WebOfScience
```

Comment:
Forwardable?  I prefer that it does not rely on rails magic for this to work.  The less rails magic the better in the business logic of the app.

Reply:
This is a rails app.  We rely on railsisms **throughout** the application, including the parts you imagine are now or someday could be separate.  I am disinclined to include additional dependencies in places where a perfectly functional (and more commonly used) solution is already available and, as demonstrated here, works perfectly well.  

If you want to push the issue, I will spin off a PR to remove every use of Forwardable.